patches/react-native+0.73.4+014+fixIOSWebViewCrash.patch

```diff
@@ -0,0 +1,57 @@
+From 03e1b9910a795c9e3cf0e2f09f9e4590eeabca29 Mon Sep 17 00:00:00 2001
+From: Riccardo Cipolleschi <riccardo.cipolleschi@gmail.com>
+Date: Mon, 19 Feb 2024 11:28:04 +0000
+Subject: [PATCH] [RN][iOS] Fix compiler flags passed to libraries
+
+---
+ .../scripts/cocoapods/__tests__/new_architecture-test.rb    | 5 +++--
+ packages/react-native/scripts/cocoapods/new_architecture.rb | 6 +++++-
+ 2 files changed, 8 insertions(+), 3 deletions(-)
+
```

Comment:
I _think_ all this shouldn't be here? AFAIK those should be typical `.patch` files that just contain a diff.

```suggestion
```

Reply:
Sorry @francoisl this was supposed to be a draft. I'm failing to compile the IOS app locally so I gave this a try. I got the `.patch` file generated by github from a PR:

https://patch-diff.githubusercontent.com/raw/facebook/react-native/pull/43088.patch